ma_gym/envs/pong_duel/pong_duel.py

```diff
@@ -32,6 +33,11 @@ def __init__(self, step_cost=0, reward=1, max_rounds=10):
         self.ball_pos = None
         self.__rounds = None
 
+        # agent pos(2), ball pos (2), balldir (6-onehot)
+        self.obs_low = np.array([0., 0., 0., 0.] + [0.] * len(BALL_DIRECTIONS))
+        self.obs_high = np.array([1., 1., 1., 1.] + [1.] * len(BALL_DIRECTIONS))
+        self.observation_space = MultiAgentObservationSpace([spaces.Box(self.obs_low, self.obs_high)for _ in range(self.n_agents)])
```

Comment:
Let's make obs_low and obs_high private attributes.

Reply:
Thanks. Addressed